packages/zoe/src/objArrayConversion.js

```diff
@@ -70,6 +70,21 @@ export const filterObj = /** @type {function<T>(T, string[]): T} */ (
   return newObj;
 };
 
+// return a new object with only the keys in subsetKeywords. `obj`
+// may *not* have all the `subsetKeywords`.
```

Comment:
Sorry, I read "may *not* have all" as "is not allowed to have all", while I think it should say "is allowed to not include some".  "might *not* have all" is also unambiguous.

Reply:
Sure, that sounds good